frontend/src/index.tsx

```diff
@@ -29,12 +28,10 @@ const container = document.getElementById('root') as HTMLElement
 const root = createRoot(container)
 root.render(
   <BrowserRouter>
-    <StrictMode>
```

Comment:
I get that `<StrictMode>` was causing the embed image feature to not work locally, but I feel like removing it entirely from the entire application is not very good.

Have [read this](https://react.dev/reference/react/StrictMode) and seems like `StrictMode` could help catch problems in the future. Unfortunately, there is no way to opt out of Strict Mode inside a tree wrapped in `StrictMode`.

Perhaps a compromise solution might be to:
- Remove `StrictMode` for the purposes of this PR
- Add to our frontend tech debt backlog with a view towards re-adding `StrictMode` when we want to properly refactor our frontend